utils/constants.ts

```diff
@@ -13,6 +13,7 @@ export const STANDARD_NATURAL_UNIT = ether(1);
 export const STANDARD_QUANTITY_ISSUED: BigNumber = ether(10);
 export const UNLIMITED_ALLOWANCE_IN_BASE_UNITS = new BigNumber(2).pow(256).minus(1);
 export const ZERO: BigNumber = new BigNumber(0);
+export const AUCTION_TIME_INCREMENT = new BigNumber(30);
```

Comment:
What does time denote in these tests? Blocks? Or unix time

Reply:
unix seconds. Like it does on the blockchain.